scripts/zkai_attention_kv_d16_two_head_seq32_fused_softmax_table_native_gate.py

```
@@ -0,0 +1,1144 @@
+        str(tmp_path),
+    ]
+    try:
+        completed = subprocess.run(
+            command,
+            cwd=ROOT,
+            text=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            timeout=FUSED_VERIFY_TIMEOUT_SECONDS,
+            check=False,
+        )
```

Ruff (0.15.13)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)